core/services/vrf/listener_v2.go

```diff
@@ -764,9 +764,10 @@ func (lsn *listenerV2) processRequestsPerSub(
 					EncodedPayload: hexutil.MustDecode(p.payload),
 					GasLimit:       p.gasLimit,
 					Meta: &txmgr.EthTxMeta{
-						RequestID: &requestID,
-						MaxLink:   &maxLinkString,
-						SubID:     &p.req.req.SubId,
+						RequestID:     &requestID,
+						MaxLink:       &maxLinkString,
+						SubID:         &p.req.req.SubId,
+						RequestTxHash: &p.req.req.Raw.TxHash,
```

Comment:
Can you also populate `RequestTxHashes` in https://github.com/smartcontractkit/chainlink/blob/develop/core/services/vrf/listener_v2_types.go#L155 as well to have this for batch transactions?

You'll have to add a `requestTxHashes` field to the `batchFulfillment` type to track these.